src/test/java/reactor/netty/http/server/HttpServerTests.java

```diff
@@ -794,6 +795,7 @@ public void testConnectionCloseOnServerError() {
 		FutureMono.from(ch.get().closeFuture()).block(Duration.ofSeconds(30));
 	}
 
+	@Ignore
```

Comment:
isn't that a pretty important feature @violetagg @OlegDokuka @smaldini ?

Reply:
#1208 fixes that. But it is a separate PR, so this one goes with the ignored test